backend/app/api/v1/users.py

```
@@ -0,0 +1,26 @@
+
+from app.core.database import get_db
+from app.core.security import create_access_token
+ 
+router = APIRouter(prefix="/users", tags=["users"])
+   
+@router.post("/login", response_model=TokenOut, status_code=status.HTTP_200_OK)
+async def login(user_data: UserLogin, db: AsyncSession = Depends(get_db)):
+    try:
+        user = await user_service.login_user(db, user_data)
+
+    except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)